langchain4j-oracle/src/test/java/dev/langchain4j/data/document/loader/OracleDocumentLoaderTest.java

```
@@ -0,0 +1,89 @@
+package dev.langchain4j.data.document.loader;
+
+import dev.langchain4j.data.document.loader.oracle.OracleDocumentLoader;
+import dev.langchain4j.data.document.Document;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+import org.slf4j.LoggerFactory;
+
+import io.github.cdimascio.dotenv.Dotenv;
+
+public class OracleDocumentLoaderTest {
+
+    private static final org.slf4j.Logger log = LoggerFactory.getLogger(OracleDocumentLoaderTest.class);
+
+    Dotenv dotenv;
+    OracleDocumentLoader loader;
+
+    @BeforeEach
+    void setUp() {
+        dotenv = Dotenv.configure().load();
+
+        try {
+            Connection conn = DriverManager.getConnection(
+                    dotenv.get("ORACLE_JDBC_URL"), dotenv.get("ORACLE_JDBC_USER"), dotenv.get("ORACLE_JDBC_PASSWORD"));
+            loader = new OracleDocumentLoader(conn);
+        } catch (SQLException ex) {
```

Instead of all these catchs that might swallow some of the details we can just declare them (void setUp() throws SQLException). Then you don't need to catch the checked exception.